assets/js/base/context/editor/index.js

```diff
@@ -23,11 +25,16 @@ export const useEditorContext = () => {
 /**
  * Editor provider
  *
- * @param {Object}  props                 Incoming props for the provider.
- * @param {*}       props.children        The children being wrapped.
- * @param {number}  [props.currentPostId] The post being edited.
+ * @param {Object}   props                 Incoming props for the provider.
+ * @param {*}        props.children        The children being wrapped.
+ * @param {CartData} [props.previewCart]   The preview data for editor.
+ * @param {number}   [props.currentPostId] The post being edited.
  */
-export const EditorProvider = ( { children, currentPostId = 0 } ) => {
+export const EditorProvider = ( {
+	children,
+	currentPostId = 0,
+	previewCart,
```

Comment:
Should this have a default since it's optional?

Reply:
what kind of default are we hoping for? an empty object or the cart skeleton?

Reply:
I went with a basic skeleton {} so that we can catch any future issue (of a component reaching for preview data when he shouldn’t).